### PR TITLE
Fix example jobs

### PR DESCRIPTION
**Pull request by jdrew82**
This PR addresses the issues in #367 and updates the example Jobs to support handling Roles, Platforms, Manufacturers, DeviceTypes, and Devices.

### CODE REVIEW

File: nautobot_ssot/jobs/examples.py [hunk 1]
```diff
@@ -94,14 +126,100 @@ class TenantModel(NautobotModel):
...
+    _attributes = ("part_number", "u_height", "is_full_depth")
+
+    model: str
+    manufacturer__name: str
+    part_number: str
+    u_height: int
+    is_full_depth: bool
+
+    # Not in _attributes or _identifiers, hence not included in diff calculations
+    pk: Optional[UUID]
```
Comment (Kircheneer) [inline nautobot_ssot/jobs/examples.py:152]: With the newest iteration of contrib, you get these out of the box, no need to explicitly annotate them - pk should be available on all `NautobotModel`s

**Review by Kircheneer — CHANGES_REQUESTED** [review]
Just one minor thing that I'd like to get fixed in case people use these as examples (huh 😄). Thanks for taking care of this!

File: nautobot_ssot/jobs/examples.py (hunk 1 above)
Comment (jdrew82) [reply nautobot_ssot/jobs/examples.py:152]: That's what I thought but it appeared to be used when communicating to the remote Nautobot instance.

Comment (Kircheneer) [reply nautobot_ssot/jobs/examples.py:152]: Ahhh, yeah got it

**Review by Kircheneer — APPROVED** [review]
LGTM, merge if you want :)

**Pull request merged**